packages/tachyons/src/core/_font-weight.css

```diff
@@ -7,39 +7,48 @@
 }
 
 .fw1 {
-    font-weight: 100;
+    font-variation-settings: "wght" 100;
```

Comment:
In the release documentation, we should specify that everywhere that we used font-weight: 600 in a css file, we should also specify the equivalent font-variation-settings: "wght" 600;

Reply:
This issue is specific to Firefox Mac, that's why I am not sure if it should be documented since FFox Mac will probably fix this down the line and we will be able to go back to font weights eventually.